custom-routes-vis/src/main/java/com/google/sps/servlets/GetDirectionsServlet.java

```diff
@@ -17,40 +17,48 @@
 import java.io.InputStream;
 import java.io.IOException;
 import java.io.OutputStream;
+import java.net.HttpURLConnection;
 import java.net.MalformedURLException;
 import java.net.URL;
 import java.net.URLConnection;
-import java.net.HttpURLConnection;
 import java.util.Scanner;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+/** Servlet that renders customized routes by calling a specified API. */
 @WebServlet("/get-directions")
 public class GetDirectionsServlet extends HttpServlet {
+  private static final String COMPUTE_ROUTES = "compute-routes";
+  private static final String COMPUTE_ROUTES_ALPHA = "compute-routes-alpha";
 
+  /** 
+   * Handles the request to get routes and responds with results from a call to a specified API. 
+   * The request specifies the origin, destination, service endpoint, and, optionally, an API
+   * key for the API request.
```

Comment:
This could be phrased more clearly
```suggestion
   * Handles the request to get routes between the origin and destination specified in the request parameters using the requested API. 
```

You would then have the @return and @param specifying what your request and response contains.

Reply:
ok